src/platform/mhv/hooks/useDatadogRum.jsx

```diff
@@ -31,13 +29,25 @@ const useDatadogRum = (config, userInfo) => {
         !window.Mocha
       ) {
         initializeDatadogRum(config);
-        if (userInfo) {
-          setRumUser(userInfo);
-        }
       }
     },
-    [config, userInfo],
+    [config],
   );
 };
 
-export { useDatadogRum };
+// REMINDER: Always be conscience of PII and Datadog
+const setDatadogRumUser = user => {
+  if (
+    // // Prevent RUM from running on local/CI environments.
+    environment.BASE_URL.indexOf('localhost') < 0 &&
+    // Only run if DD is configured.
+    window.DD_RUM?.getInitConfiguration() &&
+    // Not during unit tests
+    !window.Mocha &&
```

Comment:
since this is already conditioned in `useDatadogRum` hook, do we still need it in this setter function?

Reply:
yes, since, this could in theory be used without calling the useDataDog hook, so this in extra layer of guarding against this getting called in this scenario. 

There might be a better  way to structure this, and I am open to ideas.